SubmittedAssignments/Vrushali_Gavali-44/index.html

```
@@ -0,0 +1,73 @@
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Charm:wght@400;700&family=Cutive&family=Estonia&family=Great+Vibes&family=Metal&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./style.css">
+
+  <title>Menu Card</title>
+</head>
+<body>
+  <!-- Usage of menu-component -->
+  <menu-component
+    config='{
```

HTMLHint (1.5.0)

[error] The value of attribute [ config ] must be in double quotes.

(attr-value-double-quotes)